### ISSUE TITLE
[Moderate] The decorative image is not hidden from screen readers

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#599]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87750 |
| **Checkpoint** | 1.1 Text Alternatives › Alternative Text (Decorative Images) (1.1.1.d) |
| **Impact** | Moderate |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Import/Export Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 12:20 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The decorative image is not hidden from screen readers.
On Import Settings, if you enter an invalid URL, an Oops! screen appears with a << Go Back button, but the "less than" signs are announced by the screen reader.  See Import Export User Experience video at 4:15.
Invalid URL: https://github.com/LibraryOfCongress/bagger/blob/master/bagger-business/src/main/resources/gov/loc/repository/bagger/profiles/SANC-state-profile.json

## Recommendation

Rule : 
Images that do not convey content, are decorative, or are redundant to content that is already conveyed in text MUST be coded to be ignored by screen readers.

How To Fix: 
Fix this issue by using ONE of the following techniques:
1. Use aria-hidden="true" on the icon font.

<a href="https://www.company.com">
<span class="font-icon font-icon--home" aria-hidden="true"></span> this should be hidden
</a>

2. Turn the image into a CSS background image.

Reference: 
Invisible Content: https://webaim.org/techniques/css/invisiblecontent/
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

Background: 
Decorative images do not add information to the content of a page. Examples of decorative images include an image whose content is provided by adjacent text, or an image that is used just for visual effect such as a hero image, a background image, a stock photo, or a decorative flourish. Decorative images should be coded in such a way that screen readers can ignore them.

## Screenshots

![02_decorative](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87750/02_decorative.png)
